Formula/rdkit.rb

```diff
@@ -55,6 +60,7 @@ def install
       -DBoost_NO_BOOST_CMAKE=ON
       -DPYTHON_INCLUDE_DIR=#{py3include}
       -DPYTHON_EXECUTABLE=#{python_executable}
+      -DPYTHON_NUMPY_INCLUDE_PATH=#{numpy_include}
```

Comment:
Was the macOS build able to find `numpy` without this configuration? If it didn't, then I agree with bumping the revision. If it did, then the revision bump is not necessary.